images/buildkit/tests/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+variable "script" {
+  description = "path of the script to run, relative to this folder."
+}
+
+data "oci_exec_test" "build" {
+  digest = var.digest
+  script = var.script
+}
```

Comment:
NB: at this point it seems somewhat silly to even have a module for this 🤣

Reply:
Removed.